### ISSUE TITLE
Periodic Tasks

### ISSUE THREAD

**Issue by Enginerd0** [zariiii9003/python-can-cvector#2]
Hello,

Has this implementation also therefore increased performance of the periodic tasks? One thing I've noticed with python-can is that a larger number of periodic tasks tends to decrease performance and integrity of each of those messages sending on time. Was just curious if you had done any testing there? I have a use case where I need to have a large number of periodic tasks with counters in each message and timing is critical.

Thanks for creating this!

**Comment by zariiii9003** [comment]
No, this library won't help with that. The python-can broadcast manager starts a new thread for each message, this does not scale well. I recommend implementing your own single-threaded broadcast manager.
